### ISSUE TITLE
Upgrade from C117 -> C118 on Win x64 via 1.60.x - Nightly (Checking C118)

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#33229]
As per process, QA runs through the following cases to ensure that the major chromium bump that's about to get merged into `master` doesn't regress the `Nightly` channel. Once the major chromium bump is merged into `master`, QA will run through a full manual pass on `Windows` & `Android`.

### Startup & Components

- [ ] Verify that Brave is only contacting `*.brave.com` endpoints on first launch using either `Charles Proxy`, `Fiddler`, `Wireshark` or `LittleSnitch` (or a similar application)
  - [ ] Verify that opening a NTP doesn't trigger any outbound connections related to widgets without user interaction
- [ ] Remove the following component folders and ensure that they're being re-downloaded after restarting the browser:
  - [ ] `afalakplffnnnlkncjhbmahjfjhmlkal`: `AutoplayWhitelist.dat`, `ExtensionWhitelist.dat`, `ReferrerWhitelist.json` and `Greaselion.json`
  - [ ] `CertificateRevocation`
  - [ ] `gccbbckogglekeggclmmekihdgdpdgoe`: (Sponsored New Tab Images)
  - [ ] `gkboaolpopklhgplhaaiboijnklogmbc`: Brave Ad Block List Catalog
  - [ ] `iodkpdagapdfkphljnddpjlldadblomo`: Brave Ad Block Updater
  - [ ] `mfddibmblmbccpadfndgakiopmmhebop`: Brave Ad Block List Catalog
  - [ ] `oofiananboodjbbmdelgdommihjbkfag`: HTTPSE
  - [ ] `Safe Browsing`
- [ ] Restart the browser, load `brave://components`, wait for 8 mins and verify that no component shows any errors

**Note:** Always double check `brave://components` to make sure there's no errors/missing version numbers

### Upgrade

- [ ] Make sure that data from the last version appears in the new version OK
- [ ] Ensure that `brave://version` lists the expected Brave & Chromium versions
- [ ] With data from the last version, verify that:
  - [ ] Bookmarks on the bookmark toolbar and bookmark folders can be opened
  - [ ] Cookies are preserved
  - [ ] Installed extensions are retained and work correctly
  - [ ] Opened tabs can be reloaded
  - [ ] Stored passwords are preserved
  - [ ] Sync chain created in previous version is retained
  - [ ] Social media-blocking buttons changes are retained
  - [ ] Custom filters under brave://settings/shields/filters are retained
  - [ ] Custom lists under brave://settings/shields/filters are retained
  - [ ] Rewards
    - [ ] BAT balance is retained
    - [ ] Auto-contribute list is retained
    - [ ] Both Tips and Monthly Contributions are retained
    - [ ] Panel transactions list is retained
    - [ ] Changes to rewards settings are retained
    - [ ] Ensure that Auto Contribute is not being enabled when upgrading to a new version if AC was disabled
  - [ ] Ads
    - [ ] Both `Estimated pending rewards` & `Ad notifications received this month` are retained
    - [ ] Changes to ads settings are retained
    - [ ] Ensure that ads are not being enabled when upgrading to a new version if they were disabled
    - [ ] Ensure that ads are not disabled when upgrading to a new version if they were enabled

**Comment by MadhaviSeelam** [comment]
Verification `PASSED` using
```
Brave | 1.60.47 Chromium: 118.0.5993.21 (Official Build) nightly (64-bit)
-- | --
Revision | 8c2edac2b59451745c32ee03f6aca51abfe4d222
OS | Windows 11 Version 22H2 (Build 22621.2283)
```
And 
Verified upgrade from of CR 118 nightly from 
`Brave | 1.60.47 Chromium: 118.0.5993.21` > `1.60.50 Chromium: 118.0.5993.21  `

**Comment by MadhaviSeelam** [comment]
```
Brave | 1.60.47 Chromium: 118.0.5993.21 (Official Build) nightly (64-bit)
-- | --
Revision | 8c2edac2b59451745c32ee03f6aca51abfe4d222
OS | Windows 11 Version 22H2 (Build 22621.2283)
```


<details>
<summary>C117 --> C118 on-demand upgrade check   - `PENDING` </summary>

1.installed `1.60.44 (Chromium 117.0.5938.88 ` (latest public release) and closed Brave
2 created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url`
  * set `url` as `https://updates.brave.com/service/update2`
3 set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `cr-bump-update-tests-x64`
4. realunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
5. ensured that clicking `Relaunch` restarted the browser without any issues
6.  ensured Brave was updated to `1.59.66 Chromium: 117.0.5938.35` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|----------------|----------------|------------------
<img width="923" alt="image" src="https://github.com/brave/brave-browser/assets/98358127/b209d4e0-a6b6-4119-8445-d7b2ecd97116">|<img width="950" alt="image" src="https://github.com/brave/brave-browser/assets/98358127/17d7cc88-3079-419a-b13e-944a559a1d6c">|<img width="951" alt="image" src="https://github.com/brave/brave-browser/assets/98358127/f042caa8-1a83-4b08-b303-6b84dbe6155a">|<img width="948" alt="image" src="https://github.com/brave/brave-browser/assets/98358127/4ecca057-4735-4df1-956a-5b30f9cc6254">|<img width="644" alt="image" src="https://github.com/brave/brave-browser/assets/98358127/10c15ad1-e36c-4b12-8a7e-dd21d7c4e94f">

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Win 10 x86` using the following build(s):

```
Brave | 1.60.55 Chromium: 118.0.5993.21 (Official Build) nightly (32-bit)
-- | --
Revision | 2904d1b0dea8ee4593706e460d6fc4565d4a27ee
OS | Windows 10 Version 22H2 (Build 19045.3393)
```

<details>
<summary>C117 --> C118 on-demand upgrade check - PASSED</summary>

* installed `1.60.58 Chromium: 117.0.5938.92` and closed Brave
* set `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `cr-bump-update-tests-x86`
* realunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `1.60.55 Chromium: 118.0.5993.21` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|----------------|---------------
![image](https://github.com/brave/brave-browser/assets/2602313/fcf3a307-6805-4b4c-9a94-a0ae7ea4ad6f) | ![image](https://github.com/brave/brave-browser/assets/2602313/b9ce3882-bae2-4c06-8ede-8a03e7c50259) | ![image](https://github.com/brave/brave-browser/assets/2602313/6ba09078-0c86-43bb-924e-c117958afbcd) | ![image](https://github.com/brave/brave-browser/assets/2602313/5608eb62-2be2-4f42-b65c-ace52d607823)

</details>

<details>
<summary>C118 --> C118 on-demand upgrade check - PASSED</summary>

* installed `1.60.47 Chromium: 118.0.5993.21` via `BraveBrowserStandaloneNightlySetup32_118_1_60_47.exe` and closed Brave
* set `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `cr-bump-update-tests-x86`
* realunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `1.60.55 Chromium: 118.0.5993.21` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|----------------|---------------
![image](https://github.com/brave/brave-browser/assets/2602313/badc06f7-5ba3-41fb-83f6-fb081d50eb15) | ![updated2](https://github.com/brave/brave-browser/assets/2602313/172a9058-86f7-4b89-923c-0b6b84be2eb4) | ![image](https://github.com/brave/brave-browser/assets/2602313/191b9399-d0d9-4713-a2b4-f9b05af90bfd) | ![image](https://github.com/brave/brave-browser/assets/2602313/7f238ff1-ab0d-4cb1-b4c8-ab179063cbbf)
</details>

<details>
<summary>C117 --> C118 background upgrade check - PASSED</summary>

* installed `1.60.58 Chromium: 117.0.5938.92` and closed Brave
* set `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `cr-bump-update-tests-x86`
* started `BraveSoftwareUpdateTaskMachineUA{B6C0604F-57A6-4CDB-A0CC-2B01854EAEF0}` via `Task Scheduler`
* waited ~30min or so before launching Brave and ensured it was updated to `1.60.55 Chromium: 118.0.5993.21`
* restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background`

**`Example`** | **`Example`** | **`Example`**
----------------|----------------|---------------
![image](https://github.com/brave/brave-browser/assets/2602313/06b9ca31-f374-4875-b107-0e6822af33c0) | ![image](https://github.com/brave/brave-browser/assets/2602313/c3a6fa98-bbb2-42a0-a1a2-8834febb983c) | ![image](https://github.com/brave/brave-browser/assets/2602313/1d81b204-13ee-4823-8bc8-4f837ace02ea)

</details>

<details>
<summary>C118 --> C118 background upgrade check - PASSED</summary>

* installed `1.60.47 Chromium: 118.0.5993.21` via `BraveBrowserStandaloneNightlySetup32_118_1_60_47.exe` and closed Brave
* set `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `cr-bump-update-tests-x86`
* started `BraveSoftwareUpdateTaskMachineUA{5BF1711F-9FAE-44FB-BF1C-552959315E39}` via `Task Scheduler`
* waited ~30min or so before launching Brave and ensured it was updated to `1.60.55 Chromium: 118.0.5993.21`
* restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background

**`Example`** | **`Example`** | **`Example`**
----------------|----------------|---------------
![image](https://github.com/brave/brave-browser/assets/2602313/98192952-04c3-4b6e-85c0-6fac6b01acb4) | ![image](https://github.com/brave/brave-browser/assets/2602313/fe9f7a8d-eb57-4bb5-b0c5-c21971763d94) | ![image](https://github.com/brave/brave-browser/assets/2602313/75cd9142-299b-4d0a-8fb0-106d5f2abb30)

</details>
